packages/ui/src/components/Alert/style.less

```diff
@@ -0,0 +1,42 @@
+@import "antd/lib/alert/style/index.less";
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import (reference) "@open-condo/ui/src/components/style/mixins/typography";
+@import "./mixins";
+
+.condo-alert {
```

Comment:
Did you tried just to set this vars?) 

https://github.com/ant-design/ant-design/blob/4.x-stable/components/style/themes/default.less#L951